typesafe/shared/src/main/scala/zio/config/typesafe/TypesafeConfigDescriptor.scala

```diff
@@ -0,0 +1,9 @@
+package zio.config.typesafe
+
+import com.typesafe.config._
+import zio.config._
+
+object TypesafeConfigDescriptor {
```

Comment:
Hey could we rename this to PureConfigDescriptor ?

Reply:
@afsalthaj how about we just move the inner `configValueConfigDescriptor` into package and drop `TypesafeConfigDescriptor`?